includes/class-wpcom-legacy-redirector.php

```diff
@@ -174,12 +194,14 @@ static function insert_legacy_redirect( $from_url, $redirect_to, $validate = tru
 
 		return true;
 	}
+
 	/**
 	 * Validate the URLs
 	 *
-	 * @param string $from_url
-	 * @param string $redirect_to
-	 * * @param bool $validate          Validate $from_url and $redirect_to values.
+	 * @param string $from_url     URL to redirect (source).
+	 * @param string $redirect_to  URL to redirect to (destination).
+	 *
+	 * @return array|WP_Error      Error if invalid redirect URL specified; returns array of params otherwise.
```

Comment:
No space above, and don't line up the `@return` description with the `@param` descriptions.